packages/nx/src/project-graph/utils/project-configuration-utils.ts

```diff
@@ -791,6 +791,8 @@ export function mergeTargetConfigurations(
     );
   }
 
+  result.parallelism ??= true;
```

Comment:
I don't think we should set the default here, this code should be solely responsible for merging 2 target configurations. If we are normalizing to a default value, that should be done inside `normalizeTarget`

Reply:
done. updated